packages/blade/src/components/Button/_decisions/decisions.md

```diff
@@ -0,0 +1,81 @@
+# Button Decisions <!-- omit in toc -->
+- [Design](#design)
+- [Internal BaseButton Component](#internal-basebutton-component)
+- [Button Component](#button-component)
+- [Component Breakdown](#component-breakdown)
+- [Open Questions](#open-questions)
+
+## Design
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=5200%3A0) to all variants of the Button component
+
+## Internal BaseButton Component
+-  Internal component that exposes certain extra props to enable creation of Component Level Action Buttons like `positive`, `negative`, `neutral`
+-  **BaseButton API**:
+   -  **`variant`**: `primary` | `secondary` | `tertiary` | `link`
+      -  **default**: `primary`
+      -  Link will look like an inline Link component but will have the role of a `button`
+   -  **`intent`**: `positive` | `negative` | `notice` | `info` | `neutral`
+      -  **default**: `undefined`
+      -  We will use this to set the `intent` for internal Blade components that have positive, negative, notice, info, or neutral buttons like an `Alert` component
+   -  **`contrast`**: `high` | `low`
```

Comment:
Seems like we're not exposing this as a prop on the main Button component (for consumers). How would they configure it?

Reply:
The Button component will only have a `variant` prop that would allow you to choose the kind of button you want. `contrast` is only required when we are using it along with `intent` for internal Blade components like an `Alert` that could have a `low` as well as `high` contrast `positive` intent button. 

Our assumption is that our products will not require a standalone `low contrast positive` button (except a couple edge cases where we would encourage using custom buttons). We're doing this to ensure these different combinations of buttons don't get over-used within our products leading to breakage in the design language. 

But the fact that we have these capabilities built in the `BaseButton` itself would ensure that we _can_ expose these to our consumers when we see a massive demand for it

Reply:
Makes sense, perhaps add it as a note to the md file?